plug-ins/movement/recallmovement.cpp

```diff
@@ -52,12 +53,12 @@ bool RecallMovement::applyFightingSkill( Character *wch, SkillReference &skill )
 
     wch->pecho( "Ты долж%1$Gно|ен|на сражаться!", wch );
     chance = skill->getEffective( wch );
-    chance = 80 * chance / 100;
+    chance = 80 * chance / 100 + skill_level(*skill, ch);
```

Comment:
@fiorine what's the idea here, to add `1%` to the chance, or add `ch.modifyLevel+1%` to the chance?